modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -0,0 +1,56 @@
+= Backup and Restore Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+This topic describes how to configure backups for Anypoint Runtime Fabric. It also describes how to perform a restoration from a backup.
+
+== Prerequisites
+
+To preserve your application  in case of system failure, ensure that your Runtime Fabrics are automatically backed up. MuleSoft highly recommends configuring backups to run hourly. Your backup archive should be stored on external storage outside the Runtime Fabric cluster.
+
+== Create a Backup
+
+To create a backup, run the following command on any controller node in the cluster.
+
+----
+gravity backup /var/lib/gravity/planet/share/backup.tar.gz
+----
+
+This command creates an archive of the current system state in: `/var/lib/gravity/planet/share/backup.tar.gz`. You can pass the following optional parameters to the `gravity backup` command:
+
+* `--follow`: outputs the backup job logs to the terminal
+* `--timeout`: specifies the deadline for the backup job (e.g.: 30s, 5m, etc). Default is 20m.
+* `--log-file`: outputs logs to a file
+
+== Perform a System Restore
+
+To restore a system from your backup, use the existing Runtime Fabric cluster or create a new Runtime Fabric cluster with the same hardware (number of servers, disks, etc.) as the cluster being restored.
+
+[WARNING]
+You can not restore Runtime Fabric on a newer version of the Runtime Fabric cluster. The new installation must use the same version of Runtime Fabric installation bundle as your backup.
```

Comment:
Also, make sure runtime fabric is completely uninstalled from the current infrastructure; or make sure to stop all infrastructure supporting the existing runtime fabric before restoring the new RTF to avoid unpredicted behavior.

Reply:
Doesn’t the restore script do this already?

Reply:
@yianL we should probably add some extra code in the restore script to check this

Reply:
Restore script will try to clean up before restoring, but it won't have any effect if you're using a different cluster to restore.

Reply:
I don’t understand this. We do a “helm delete” before restoring, so why does it make a difference if the cluster is the same or different?

Reply:
Let's discuss during scrum today.